README.md

```diff
@@ -198,7 +198,7 @@ Another usecase is if we want to encode a `case class` as an array of values, ra
 ```scala
 case class Things(s: String, i: Int, b: Boolean)
 object Things {
-  implicit val decoder: json.Decoder[Things] = json.Decoder[(String, Int, Boolean)].map(Things.tupled)
+  implicit val decoder: json.Decoder[Things] = json.Decoder[(String, Int, Boolean)].map((Things.apply _).tupled)
```

Comment:
weird, this works for me on 2.13. Are you using 2.12? In that case I think I'd rather document the cleaner syntax... or a more verbose version that works on both.

Reply:
oh, I see, it's because `object Things` doesn't do `object Things extends (String, Int, Boolean) => Things`

![](https://media.giphy.com/media/Pch8FiF08bc1G/giphy.gif)

Reply:
```scala
 implicit val decoder: json.Decoder[Things] = json.Decoder[(String, Int, Boolean)].map{ case (p1, p2, p3) => Things(p1, p2, p3) }
```

Reply:
Thanks :)